packages/components/src/CollapsiblePanel/CollapsiblePanel.component.js

```diff
@@ -112,8 +112,12 @@ function CollapsiblePanelHeader(props) {
 			? headerItem.map(renderHeaderItem)
 			: renderHeaderItem(headerItem);
 
+		const selectors = Array.isArray(headerItem)
```

Comment:
Array.isArray() statement result could be stored in a boolean const to avoid duplicate calls (few lines above in the file)